demo.py

```diff
@@ -13,7 +13,7 @@ def run_demo():
     # Create an asa sh tech object
     primary_asa = ap.AsaParser(os.path.join(data_path, 'showtech_primary.txt'))
 
-
+    '''
```

Comment:
I don't think you meant to leave demo.py like this - probably you just did this for your testing and demo.  Should change so all demo code will run.

Reply:
Thank you for the reminder, I have removed the ''' marks. My apologies it had slipped my mind to put everything back once the class demo was finished.